framework/core/src/Api/Resource/AccessTokenResource.php

```diff
@@ -79,18 +79,18 @@ public function fields(): array
     {
         return [
             Schema\Str::make('token')
-                ->visible(function (AccessToken $token, Context $context) {
+                ->visible(function (#[\SensitiveParameter] AccessToken $token, Context $context) {
```

Comment:
I learned something today, good use of attributes 🙏 

I don't see a way to mark the whole class as sensitive, which - in terms of code clarity - would have been much better, especially when extensions usually don't adhere to best practices.

Reply:
Unfortunately, there is no way currently to mark whole classes as sensitive - in this case, I agree it'd be a lot cleaner, however, it's not possible right now.